tools/spaTyper/spaTyper.xml

```diff
@@ -0,0 +1,60 @@
+<tool id="spaTyper" name="Generate spa type identification" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <requirements>
+        <requirement type="package" version="0.2.1">spaTyper</requirement>
```

Comment:
To figure out why the test is failing now, when you click on the **Details** of the failed action, and then click on **Summary**

![image](https://user-images.githubusercontent.com/200517/225741868-5ca22db5-6c29-49c1-b39d-1dcc398bb5bf.png)

And from here, scroll to the bottom and click on **All tool test results** in the **Artifacts** section:

![image](https://user-images.githubusercontent.com/200517/225742030-240b16b1-3845-45fb-a4ab-c7598693e282.png)

You should get a zip file with the HTML test results. Opening this HTML file up in a browser should give you:

![image](https://user-images.githubusercontent.com/200517/225742197-6fc4bcc6-e02d-4592-b123-46d74f98d386.png)

The message:

```
spaTyper: command not found
```

Suggests to me that `spaTyper` is not being properly installed in the test environment via conda.

Looking up the package name in bioconda (https://bioconda.github.io/recipes/spatyper/README.html?highlight=spatyper#package-package%20&#x27;spatyper&#x27;)

Suggests that it's due to a typo in the package name (it should be `spatyper` and not `spaTyper`).